src/animation/AnimationMixer.js

```diff
@@ -363,11 +357,11 @@ Object.assign( THREE.AnimationMixer.prototype, {
 			var bindings = action._propertyBindings;
 
 			// increment reference counts / sort out state
-			for ( var i = 0, n = bindings.length; i !== n; ++ i ) {
+			for ( var i = 0, n = bindings.length; i !== n; ++i ) {
 
 				var binding = bindings[ i ];
 
-				if ( binding.useCount ++ === 0 ) {
+				if ( binding.useCount++ === 0 ) {
```

Comment:
We usually *do* put a space between lvalue and pre/post-inc/dec operatos, I think...